README.md

```diff
@@ -121,6 +121,9 @@ The action can be configured with the following arguments:
   control as part of the action if you wish to perform any further tasks with
   that stack.
 
+- `exclude-protected` - (optional) Do not destroy protected resources. Only
```

Comment:
```suggestion
- `exclude-protected` - (optional) Skip destroying protected resources. Only
```

This change in language disambiguates the distinction between Pulumi and Terraform, where `protect` will always prevent the destruction of the resource, and `destroy` will fail under any circumstance.